### ISSUE TITLE
Shape size for 'line' orbit traps should control line's length

### ISSUE THREAD

**Issue by dluchansky** [buddhi1980/mandelbulber2#775]
##### System information (version)

- mandelbulber2 => 2.21
- Operating System / Platform => Windows 10 64bit

##### Detailed description

It would seem, from my personal testing, that the shape size parameter for line-type orbit trap lights has no effect.
This parameter should be able to control the line's length, and maybe 0 or -1 should set it to the current behavior (of an infinite-length line.)
##### Steps to reproduce
Use any orbit trap lights with line shape.

**Comment by buddhi1980** [comment]
You are right. It has no effect. It is a good idea to use it to define length of the line.

**Comment by dluchansky** [comment]
When upgrading, don't forget to change interpret all existing formulas using line orbit traps to set the length to infinite. It could break some files otherwise.

**Comment by mclarekin** [comment]
a) alternatively a line can be viewed as a cylinder with a very small radius
b) a line of fixed length has orbittrap distances at right angles along the length and radial orbitrap distances past the ends, For similar maths  https://www.youtube.com/watch?v=PMltMdi1Wzg

**Comment by mclarekin** [comment]
a) clarify, size could be radius instead of length, then we have an infinte tube

General, tube or cylinder are quite cheap, but there is a lot more orbittrap lights we could implement but they are more expensive and i would consider them  very low on the programs  TODO priority list
